### PR TITLE
MEN-3637 identity attributes count test update.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: backend-tests/tests/test_inventory.py [hunk 1]
```diff
@@ -277,13 +277,13 @@ def test_ok(self, user):
 
             api_dev = r.json()
             # Expected inventory count per scope:
-            # {"inventory": 3, "identity": 1, "system": 2}
-            assert len(api_dev["attributes"]) == 6
+            # {"inventory": 3, "identity": 1+2, "system": 2} # +2 comes from the id_data see MEN-3637
+            assert len(api_dev["attributes"]) == 8
```
Comment (lluiscampos) [inline backend-tests/tests/test_inventory.py:281]: I don't understand why the test passed in https://github.com/mendersoftware/deviceauth/pull/351 and then failed in master.

File: backend-tests/tests/test_inventory.py [hunk 2]
```diff
@@ -277,13 +277,13 @@ def test_ok(self, user):
...
             # new scopes: identity and system holding authset status and
             #             time-stamp values respectively
 
             for a in api_dev["attributes"]:
-                if a["name"] == "mac":
+                if a["name"] == "mac" && a["scope"] == "inventory":
```
Comment (lluiscampos) [inline backend-tests/tests/test_inventory.py:286]: Mmm, then we are not checking any more `mac` for other scopes? Also, this change is silent (i.e. no mention in the commit) so I don't really understand the context.

File: backend-tests/tests/test_inventory.py (hunk 1 above)
Comment (merlin-northern) [reply backend-tests/tests/test_inventory.py:281]: I think it is because 351 ran without inventory changes merged in.

Comment (lluiscampos) [reply backend-tests/tests/test_inventory.py:281]: That could be, but then the inventory changes PR should have failed...

Anyway, for the next time we should use a feature branch for these kind of multi repos changes.

File: backend-tests/tests/test_inventory.py (hunk 2 above)
Comment (lluiscampos) [reply backend-tests/tests/test_inventory.py]: @merlin-northern Thank you for a beautiful commit message :heart:

**Pull request merged**